library/core/src/future/ready.rs

```diff
@@ -24,6 +24,29 @@ impl<T> Future for Ready<T> {
     }
 }
 
+impl<T> Ready<T> {
+    /// Consumes the `Ready`, returning the wrapped value.
+    ///
+    /// # Panics
+    ///
+    /// Will panic if this [`Ready`] was already polled to completion.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use std::future;
+    ///
+    /// let a = future::ready(1);
+    /// assert_eq!(a.into_inner(), 1);
+    /// ```
+    #[stable(since = "1.65.0", feature = "dummy")]
```

Comment:
I believe this is how it's done when it's supposed to be insta-stable?

Reply:
I don't think this should be insta stable, this cam start out unstable just fine.

Reply:
Done. I used `none` as issue, how does this work, should I create a tracking issue now?

Reply:
Does it need an ACP first?

Reply:
I asked on Zulip if this needs an RFC or anything else, @joshtriplett said it's fine.
https://rust-lang.zulipchat.com/#narrow/stream/219381-t-libs/topic/.60Ready.3A.3Ainto_inner.28.29.60

Reply:
Please do file a tracking issue. Otherwise looks good.

Reply:
Done https://github.com/rust-lang/rust/issues/101196.